notebook_utils/berdl_notebook_utils/spark/database.py

```diff
@@ -96,32 +103,33 @@ def create_namespace_if_not_exists(
 
     if append_target:
         try:
-            namespace, db_location = generate_namespace_location(namespace, tenant_name)
-        except Exception as e:
-            print(f"Error creating namespace: {e}")
-            raise e
+            delta_namespace, db_location = generate_namespace_location(namespace, tenant_name)
```

Comment:
changed this variable to `delta_namespace` so that it is clearer that it's the governance service / minio-approved namespace, not the raw string entered by the user.